File: Molinos.Scato.WebPuerto/src/app/shared/modelos/administracion/concepto.ts
export class Concepto {
    id: number;
    descripcion: string;
    tipoConcepto: TipoConcepto;
    moneda: Moneda;
    tipoTarifa: TipoTarifa;
    presentaAjuste: boolean;
    porProducto: boolean;
    porEmbarque: boolean;
}

export class TipoConcepto {
    id: number;
    descripcion: string;
}

export class Moneda {
    id: number;
    descripcion: string;
}

export class TipoTarifa {
    id: number;
    descripcion: string;
}

File: Molinos.Scato.Dominio/Entidades/Administracion/Concepto.cs
using System.ComponentModel.DataAnnotations;

namespace Molinos.Scato.Dominio.Entidades
{
    public class Concepto : IIdentificable
    {
        [Key]
        public virtual int Id { get; set; }
        public virtual string Descripcion { get; set; }
        public virtual TipoConcepto TipoConcepto { get; set; }
        public virtual Moneda Moneda { get; set; }
        public virtual TipoTarifa TipoTarifa { get; set; }
        public virtual bool PresentaAjuste { get; set; }
        public virtual bool PorProducto { get; set; }
        public virtual bool PorEmbarque { get; set; }
    }
}